pages/success.js

```diff
@@ -1,22 +1,26 @@
 const { I } = inject();
 
 module.exports = {
+  
   h1 : {xpath: '//*[@id="content"]/h1'},
-  contineButton : {xpath: '//*[@id="content"]/div/div/a'},
+  //contineButton : {xpath: '//*[@class="btn btn-primary"]'},
   thankText : {xpath: '//*[@id="content"]/p[1]'},
 
-  checkPage() {
-    const SuccessText = 'Your Account Has Been Created!';
-    const ThankText = 'Congratulations! Your new account has been successfully created!';
+  checkPage(USER) {
+    let SuccessText = '';
+    let ThankText = '';
+    if (USER) {
+      SuccessText = 'Your Account Has Been Created!';
```

Comment:
use small first letter for variables or capital, but keep it constant across the code. Only exception is for constants, e.g. `SUCCESS_TEXT`